js_modules/dagster-ui/packages/ui-core/src/gantt/RunGroupPanel.tsx

```diff
@@ -96,7 +96,7 @@ export const RunGroupPanel = ({
   });
 
   return (
-    <SidebarSection title={runs[0] ? `${runs[0].pipelineName} (${runs.length})` : ''}>
+    <SidebarSection title={runs[0] ? `Retries (${runs.length - 1})` : ''}>
```

Comment:
What do you think of the update to the number that's shown next to the title? It's currently the number of runs in the run group, which contains the original run. Renaming the section to `Retries` makes me think the number should be the number of retries, not the total number of runs in the group. However, the list of runs in the section would be the whole run group, so maybe it's more confusing to have `Retries (n)` but then see `n+1` runs in the list

Reply:
For consistency it might be best to match whatever is in the retries tag - which I believe in this case is `n`

Reply:
I think it is weird to show a number and then have the number of things not match that number (the other lists just below it match their description), but it is also weird to call it Retries and then have it not match the number of retries.

It's also maybe a little weird to say "Retries" when the run you're looking at is at the bottom of the stack? Since none of them will be retries of this run

is there a more clear way to say 'Run group'?

Reply:
... should it just say Runs?

Reply:
> It's also maybe a little weird to say "Retries" when the run you're looking at is at the bottom of the stack? Since none of them will be retries of this run

that's a good point

Reply:
spitballing:
- run lineage 
- retry lineage 
- retry group

Reply:
maybe worth a post in #eng-naming? "Run Lineage" seems ok to me